projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+func main() {
+	c := http.Client{Timeout: time.Duration(1) * time.Second}
+
+	url := "http://localhost:8080"
+	max_tries := 5
+
+	fmt.Println("----------------------------")
+
+	for i := 0; i < max_tries; i++ {
+		resp, err := c.Get(url)
+		if err != nil {
+			fmt.Println("Error from request", err)
+			return
```

Comment:
What exit code will this exit with? What does that signal to the runner of the program? (Same question for every place this function halts execution, either by a call to `return` or because the end of the function is reached)